src/sagemaker/session.py

```diff
@@ -4161,6 +4175,15 @@ def account_id(self) -> str:
         )
         return sts_client.get_caller_identity()["Account"]
 
+    def _intercept_create_request(self, request: typing.Dict, create):
+        """This function intercepts the create job request
```

Comment:
Whats the change we are trying to accomplish in this module?

Reply:
Unlike normal SM session, under pipeline session, we need to capture the create job request, and stop. This function will be overrode in `PipelineSession` class to achieve this.

Reply:
sure